docs/user-guide/import-certificates.md

```diff
@@ -0,0 +1,176 @@
+   
+   `$ sudo update-ca-certificates`
+
+3. Verify that the new expected certificate was added (the newest will be at the bottom of the list which contains an extended list of concatenated CAs)
+   
+   `$ cat /etc/ssl/certs/ca-certificates.crt`
+
+4. Run a basic curl HTTPS request from the command line. For example, run the following command:
+
+```
+  curl --request 'GET'
+--url 'https://tvt6092.svl.ibm.com:7554/jobs/api/v1?owner=ibmuser&prefix=*'
+--header 'Authorization: Basic ************'
+```
+
+## Import an existing JCERACFKS certificate
+
+To import a JCERACFKS certificate, use the example yaml according to [Scenario 4: Use a z/OS keyring-based keystore and connect to an existing certificate](#certificate-configuration-scenarios.md/#scenario-4-use-a-zos-keyring-and-connect-to-an-existing-certificate) in the article Certificate configuration scenarios.
+
+To use a JCERACFKS certificate, specify the following parameters in the `zowe.yaml` file:
+
+| Parameter | Description |
+| --------- | ------------|
+| `zowe.setup.certificate.keyring.connect.user` | This is a required parameter that specifies the owner of existing certificate. This field can have value of SITE or a user ID.|
+| `zowe.setup.certificate.keyring.connect.label` | This is a required parameter that sets the label of an existing certificate. |
+
+**Configure `zowe.yaml`:**
+```
+zowe:
+  setup:
+    certificate:
+      type: JCERACFKS
+      keyring:
+        name: ZoweKeyring
+        connect:
+          user: IBMUSER
+          label: ""
+      importCertificateAuthorities:
+        - ""
+```
+
+**Note:**  
+Due to the limitation of the RACDCERT command, the `importCertificateAuthorities` field can contain a maximum of two entries.
+
+## Import a certificate stored in an MVS data set into a Zowe key ring
+
+To import a certificate that is stored in a data set into a key ring, configure the zowe.yaml according to the example yaml in [Scenario 5: Use a z/OS keyring-based keystore and import a certificate stored in a data set](#certificate-configuration-scenarios/#scenario-5-use-a-zos-keyring-and-import-a-certificate-stored-in-a-data-set)
+
+ To use a JCERACFKS certificate, specify the following parameters in the `zowe.yaml` file.
+
+| Parameter | Description |
+| --------- | ----------- |
+|`zowe.setup.certificate.keyring.connect.dsName` | This is a required parameter which specifies the data set where the certificate stored.|
+|`zowe.setup.certificate.keyring.connect.password` | This parameter specifies the password when importing the certificate. |
+| `zowe.setup.certificate.keyring.label`  | This parameter specifies that label of the certificate that is imported. |
+
+**Configure `zowe.yaml`:**
+```
+zowe:
+  setup:
+    certificate:
+      type: JCERACFKS
+      keyring:
+        name: ZoweKeyring
+        label: localhost  # Optional, default value is localhost.
+      import:
+        dsName: ""
+        password: ""
+```
+
+<!-- Any command to run? -->
```

Comment:
Question: Does a command need to be executed?

Reply:
i didn't touch this file, but will note that the entire purpose of `zowe.setup.certificate` is to populate information that will be used by `zwe init`'s subcommand `zwe init certificate` https://docs.zowe.org/stable/appendix/zwe_server_command_reference/zwe/init/zwe-init-certificate
so that's probably what they meant to say.

Reply:
I added this line:
"The configuration of `zowe.setup.certificate` populates information to be used by the subcommand `zwe init certificate` of `zwe init`. "

Please review for accuracy.